src/utils/db.js

```diff
@@ -0,0 +1,9 @@
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  dialect: 'postgres',
+});
```

Comment:
Ensure that all environment variables (DB_HOST, DB_USER, DB_PASSWORD, DB_DATABASE) are correctly set and validated. Consider adding validation logic to check for their presence and correctness.